### PR TITLE
fix: Add schema name to Lineage graph node labels

### PR DESCRIPTION
**Pull request by ozandogrultan**
Signed-off-by: Ozan Dogrultan <ozan.dogrultan@deliveryhero.com>

### Summary of Changes

Introducing the same changes introduced [here](https://github.com/amundsen-io/amundsen/pull/1721) by [aorozco05](https://github.com/aorozco05)

### Tests

No tests are affected by this change.

### Documentation

No documentation changes are needed.

### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [x] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
    - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [x] PR includes a summary of changes.
- [x] PR adds unit tests, updates existing unit tests, __OR__ documents why no test additions or modifications are needed.
- [x] In case of new functionality, my PR adds documentation that describes how to use it.
    - All the public functions and the classes in the PR contain docstrings that explain what it does

### CODE REVIEW

File: frontend/amundsen_application/static/js/components/Lineage/Graph/chart.ts [hunk 1]
```diff
@@ -327,7 +327,11 @@ export const buildNodes = (g, targetNode, nodes, onClick) => {
     .append('text')
     .attr('dy', NODE_LABEL_Y_OFFSET)
     .attr('text-anchor', 'middle')
-    .text((d, idx) => (idx !== 0 && d.data.data.name ? d.data.data.name : ''));
+    .text((d, idx) =>
+      idx !== 0 && d.data.data.name
+        ? d.data.data.schema + '.' + d.data.data.name
+        : ''
+    );
```
Comment (Golodhros) [inline frontend/amundsen_application/static/js/components/Lineage/Graph/chart.ts:334]: I would extract a function from this, maybe 'getNodeName'

Comment (ozandogrultan) [reply frontend/amundsen_application/static/js/components/Lineage/Graph/chart.ts]: Good suggestion, I've extracted it into `getNodeLabel` function

File: frontend/amundsen_application/static/js/components/Lineage/Graph/chart.ts [hunk 2]
```diff
@@ -327,7 +335,7 @@ export const buildNodes = (g, targetNode, nodes, onClick) => {
     .append('text')
     .attr('dy', NODE_LABEL_Y_OFFSET)
     .attr('text-anchor', 'middle')
-    .text((d, idx) => (idx !== 0 && d.data.data.name ? d.data.data.name : ''));
+    .text((d, idx) => getNodeLabel(d, idx));
```
Comment (Golodhros) [inline frontend/amundsen_application/static/js/components/Lineage/Graph/chart.ts:338]: Nit, you don't need to pass again the arguments, you can do:
```suggestion
    .text(getNodeLabel);
```

**Review by Golodhros — APPROVED** [review]
LGTM

**Review by Golodhros — APPROVED** [review]

**Pull request merged**